src/api/routers/chat.py

```
@@ -117,51 +123,91 @@ async def websocket_chat(websocket: WebSocket):
-            if not message:
-                await websocket.send_json({"type": "error", "message": "Message is required"})
-                continue
-
-            logger.info(
-                f"Chat request: session={session_id}, "
-                f"message={message[:50]}..., rag={enable_rag}, web={enable_web_search}"
+        llm_config = get_llm_config()
+        api_key = llm_config.api_key
+        base_url = llm_config.base_url
+        api_version = getattr(llm_config, "api_version", None)
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)